src/Resources/config/listener.yml

```diff
@@ -161,3 +161,11 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
             - { name: kernel.event_listener, event: kernel.response, method: onKernelResponse }
+
+    contao.listener.csrf_token_cookie:
+        class: Contao\CoreBundle\EventListener\CsrfTokenCookieListener
+        arguments:
+            - "@contao.csrf.token_storage"
+        tags:
+            - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 20 }
+            - { name: kernel.event_listener, event: kernel.response, method: onKernelResponse, priority: 0 }
```

Comment:
you don't need to set a priority if it's `0`.

Reply:
Done in d874349ecc242ae71e3fc096da15369a9c281aa3